scss/_containers.scss

```diff
@@ -44,4 +38,25 @@
       }
     }
   }
+
+  // OUDS mod
+  // scss-docs-start containers-max
+  .max-width-public-website {
```

Comment:
TBH, this naming doesn't mean anything, as everything is a public website, except "private" websites.